src/index.html

```diff
@@ -12,7 +12,48 @@
       href="./styles/index.scss"
```

Comment:
The `href` attribute should point to a `.css` file, not a `.scss` file. SCSS files are preprocessed into CSS, which browsers can understand. Ensure that your build process compiles SCSS to CSS and reference the CSS file here.